tests/typescript/src/generators.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from 'blockly-test/core';
+import {JavascriptGenerator} from 'blockly-test/javascript';
+import {PhpGenerator, phpGenerator, Order} from 'blockly-test/php';
+import {LuaGenerator} from 'blockly-test/lua';
+import {PythonGenerator} from 'blockly-test/python';
+import {DartGenerator} from 'blockly-test/dart';
+
+JavascriptGenerator;
+PhpGenerator;
+LuaGenerator;
+PythonGenerator;
+DartGenerator;
+
+class TestGenerator extends PhpGenerator {}
+
+const testGenerator = new TestGenerator();
+
+testGenerator.forBlock['test_block'] = function (
+  _block: Blockly.Block,
+  _generator: TestGenerator,
+) {
+  return ['a fake code string', Order.ADDITION];
```

Comment:
I'm slightly sad that this isn't a type error, since this is PHP's `Order`, which is not compatible with the other four generators.

Reply:
What would be the type error? Php has an `ADDITION` member in its enum.

Reply:
Never mind; I had misread and somehow imagined that this code was in a loop that was doing this for all of the different languages, not just PHP.  (🍳→👨,  🤦‍♂️)

(Why PHP, though?)

Reply:
I just wanted to test one since I intend to delete these after yours are merged, and PHP seemed the most likely for me to have messed up the capitalization on :D